assets/witnet.settings.js

```diff
@@ -1,6 +1,7 @@
-import { createRequire } from "module";
-const require = createRequire(import.meta.url);
-const packageJson = require("../package.json")
+const packageJson = await import("./package.json", {
```

Comment:
We tried this a few months ago but it wasn't working. Is it possible to leave it as it is?